lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -4848,11 +4848,14 @@ DefConstructor('\shortstack[]{}  OptionalMatch:* [Dimension]',
   beforeDigest => sub { reenterTextMode();
     # then RE-RE-define this one!!!
     Let("\\\\", '\@shortstack@cr');
+    AssignRegister('\baselineskip' => Glue('-1pt'));
+    AssignRegister('\lineskip'     => Glue('3pt'));
     $_[0]->bgroup; },
-  afterDigest => sub { $_[0]->egroup; },
+  afterDigest => sub { $_[1]->getSize;    # precompute while binding in effect
```

Comment:
is the expectation that `getSize` will save that precomputed number somewhere? A tricky behavior for something named `get*`. Maybe `cachedGetSize` or `precomputeSize` or... something similar would be a little more transparent.

Reply:
Exactly, it's cached. But the lazy eval (w/caching) is something that probably needs a rethink. The point here is to compute while `\baselineskip` is bound.